build.rs

```diff
@@ -0,0 +1,8 @@
+extern crate pkg_config;
+
+fn main() {
+    #[cfg(not(feature = "dlopen"))]
+    if pkg_config::Config::new().find("xkbcommon").is_ok() {
+        return;
+    }
```

Comment:
What is the point of this `if` block? Shouldn't the build script panic if `xkbcommon` cannot be found instead?

Reply:
I tried to avoid to break existing and working behaviour: just calling `.unwrap()` would panic the build if `xkbcommon.pc ` (the configuration file used by pkg-config) isn't found but `libxkbcommon.so` exists.

But I am unaware if such configuration exists or if it is just an ill-case.

Reply:
after looking at xkbcommon repository, it seems pkgconfig is generated since at least 4 years (introduction of meson.build). so it should be an ill-case.

I will use `unwrap()`, and it should also solve the problem of build with 1.42 (where conditionnal block on conditionnal isn't supported).